app/code/local/Test/Config/etc/system.xml

```diff
@@ -0,0 +1,422 @@
+                        </source_2>
+                        <result>
+                            <label>Result</label>
+                            <frontend_type>text</frontend_type>
+                            <depends>
+                                <condition operator="or">
+                                    <source_1>1</source_1>
+                                    <source_2>1</source_2>
+                                </condition>
+                            </depends>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                        </result>
+                    </fields>
+                </or_test>
+                <xor_test>
+                    <label>XOR Test</label>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <comment><![CDATA[
+                    <h3>Result field will show if:</h3>
+                    <code>exactly one source == "Yes"</code>
+                    <br>
+                    ]]></comment>
+                    <fields>
+                        <source_1>
+                            <label>Source 1</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                        </source_1>
+                        <source_2>
+                            <label>Source 2</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                        </source_2>
+                        <source_3>
+                            <label>Source 3</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                        </source_3>
+                        <result>
+                            <label>Result</label>
+                            <frontend_type>text</frontend_type>
+                            <depends>
+                                <condition operator="xor">
```

Comment:
brainstorm: what if instead of `<condition operator="xor">` or would be `<xor>`? I mean, I don't want to change everything now, it's to evaluate an idea

Reply:
So I originally did have it be `<XOR>` etc, but `<condition operator="xor">` felt more XML-ish. Especially if we ever created any sort of schema validation.

But also, it couldn't be lowercase tags because that has a higher chance of conflicting with an actual fieldname. For example:
```xml
<depends>
    <not>1</not>
</depends>
```

Would take more effort to parse if we're looking at a config field like `general/web/not` or we're trying to evaluate a condition. So originally I thought to just use uppercase tags, but visually it doesn't look great.

Instead, conditions are **only** evaluated if the tagname is "condition" and it has an "operator" attribute, i.e. `<condition operator="not">`. If you wrote just `<condition>` it would look for a field name called "condition".

I figured this was the best option to prevent any backwards compatibility issues.

Reply:
spot-on as usual